tests/dummy/app/templates/index.hbs

```diff
@@ -1,5 +1,7 @@
 {{docs-hero}}
 
 <div class="uk-padding">
-  {{cf-form documentId=model}}
+  {{#cf-content documentId=model as |content|}}
+    {{content.form}}
```

Comment:
I think we can just use the default template, the block form is not really needed:

`{{cf-content documentId=model}}`

Reply:
Ah nice :sunglasses: